plugins/source/aws/client/testing.go

```diff
@@ -16,9 +16,14 @@ import (
 
 type TestOptions struct {
 	TableOptions tableoptions.TableOptions
+	Region       string
 }
 
 func AwsMockTestHelper(t *testing.T, table *schema.Table, builder func(*testing.T, *gomock.Controller) Services, testOpts TestOptions) {
+	if testOpts.Region == "" {
+		testOpts.Region = "us-east-1"
+	}
+
```

Comment:
This is necessary because `networkmanager` is the first service that is not available in `us-east-1` it is only available in `us-west-2`